### ISSUE TITLE
PHP 7 support

### ISSUE THREAD

**Issue by wizonesolutions** [hashbangcode/vlad#328]
@philipnorton42 follow-up to https://github.com/hashbangcode/vlad/issues/242, since PHP 7 has been out for a bit now.

**Comment by philipnorton42** [comment]
Yes. Time to get my arse in gear and include support for PHP7 :+1:

**Comment by wizonesolutions** [comment]
poke :)

**Comment by wizonesolutions** [comment]
Hmm, you still need help with this? Any pointers on what I have to change?

**Comment by Benjen** [comment]
Given that Ubuntu 16.04 includes PHP 7 by default, it would be great if VLAD could be updated to support it.

**Comment by philipnorton42** [comment]
Help on this would be great. I've been busy doing work related things for the past few months and so haven't had the time I would like to devote to the project.

As for a place to start the PHP role (at vlad_guts/playbooks/roles/php) is probably the best bet here. Ubuntu 16.04 is an TLS release it's probably a good idea to support that as well. Although that might be a different issue here. Maybe...

**Comment by philipnorton42** [comment]
I have added the ability to provision the box with PHP7. I've not tested this yet as I've been concentrating on getting the core PHP 5.6 build stable. I would appreciate some input on the current dev branch if anyone has time.
I'll keep plugging away on getting things usable.
